AxonDeepSeg/params.py

```diff
@@ -23,9 +23,22 @@
 
 # morphometrics file suffix name
 morph_suffix = Path('axon_morphometrics.xlsx')
+morph_agg_suffix = Path('subject_morphometrics.xlsx')
```

Comment:
This suffix, as well as the other morphometrics suffixes, don't adhere to the same pattern as the image suffixes as they don't include `_` prior. I think this should be changed, as the following code generated the incorrect filenames:

```python
        for axonmyelin_file in axonmyelin_files:
            morph_file = subject / axonmyelin_file.name.replace(
                str(axonmyelin_suffix), str(morph_suffix)
            )
            if not morph_file.exists():
                logger.warning(
                    f"File {morph_file} does not exist. Please generate the morphometrics file first."
                )
                return
```

Could be done in a new issue though, since this the changes required may be out of scope for this PR